cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityPrivilegesTest.scala

```diff
@@ -16,16 +16,18 @@ class SecurityPrivilegesTest extends DocumentingTest with QueryStatisticsTestSup
     initQueries(
       "CREATE USER jake SET PASSWORD 'abc123' CHANGE NOT REQUIRED SET STATUS ACTIVE",
       "CREATE ROLE regularUser",
+      "CREATE ROLE noopUser",
       "GRANT ROLE regularUser TO jake",
-      "GRANT ACCESS ON DATABASE neo4j TO regularUser"
+      "GRANT ACCESS ON DATABASE neo4j TO regularUser",
+      "DENY ACCESS ON DATABASE neo4j TO noopUser"
     )
     synopsis("This section explains how to use Cypher to manage privileges for Neo4j role-based access control and fine-grained security.")
     p(
       """
         |Privileges control the access rights to graph elements using a combined whitelist/blacklist mechanism.
         |It is possible to grant access, or deny access, or both.
         |The user will be able to access the resource if they have a grant (whitelist) and do not have a deny (blacklist) relevant to that resource.
-        |If there are no read-privileges provided at all, then the user will be denied access to the entire graph, and this will generate an error.
+        |If a user was not provided with the access privilege then access to the entire graph will be denied.
         |All other combinations of GRANT and DENY will result in the matching subgraph being visible.
```

Comment:
Sounds slightly weird to say `All other combinations of GRANT and DENY` when neither was mentioned just before.